src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -14,6 +14,19 @@
 
 @import '../variables';
 
+* {
+  font-size: 14px;
+}
+
+md-input-container {
+  margin-bottom: 5px;
```

Comment:
For width (left/right) I can see how this makes sense, but for height (top/bottom)? Am I missing something?